### PR TITLE
40 artist details page implement reading from database second part

### PR DESCRIPTION
**Pull request by NathanIsCoding**
I created some tests and implemented functionality for the details page to read the information from the database. I haven't yet been able to properly test it using Django as my branch was created before we moved the files into the new location.

### CODE REVIEW

**Review by EricSmith10 — APPROVED** [review]
looks great, thanks for planning for testing in the future, we will be able to do that.

File: code/Details.html
Comment (KenichiroAnderson) [inline code/Details.html:1]: looks good should display queried items properly in their respective places when the method is called.

File: code/unit_testing/Details.test.js
Comment (KenichiroAnderson) [inline code/unit_testing/Details.test.js:1]: nice unit testing here. smart for future issues

**Review by KenichiroAnderson — COMMENTED** [review]
all looks good here, great work!

**Pull request merged**